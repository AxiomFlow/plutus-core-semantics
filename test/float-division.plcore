(module Foo (imported) (exported () ()) (local () ())
  (define f1 (builtin divideFloat 30.0 5.0))
  (define f2 (builtin divideFloat 9.0 3.0))
  (define f3 (builtin divideFloat 20.0 3.0))
  (define f4 (builtin divideFloat 100.0 6.0)))
