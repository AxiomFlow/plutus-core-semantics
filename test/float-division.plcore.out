<T> <k> . </k> <env> f3 |-> 6.666666666666667e+00 f4 |-> 1.6666666666666668e+01 f1 |-> 6e+00 f2 |-> 3e+00 </env> </T>
